update pods cli docs (#936)
localstack/docs: content/en/user-guide/cloud-pods/pods-cli/index.md

## Commands

The following section lists the available commands for the Cloud Pods CLI.
You can have an overview of these command by typing `localstack pod --help`:

{{< command >}}
<disable-copy>
Usage: pod [OPTIONS] COMMAND [ARGS]...

  Manage the state of your instance via Cloud Pods.

Options:
  --help  Show this message and exit.

Commands:
  delete    Delete a Cloud Pod

>   inspect   Inspect the contents of a Cloud Pod This command shows the...

Reviewer: This command description looks a bit - can we remove the part with `This command shows the ...`? (both here, and in the CLI)

Also for the `versions` command below

Options:
  -p, --public               List all the available public Cloud Pods
  -f, --format [table|json]  The formatting style for the list pods command
                             output.

  --help                     Show this message and exit.
</disable-copy>
{{< / command >}}

The `list` command lists all of the available Cloud Pods. 
It shows all the pods available for a single user and its organization by default.

### `versions`

{{< command >}}
<disable-copy>
Usage: pod versions [OPTIONS] NAME

  List all available versions for a Cloud Pod

  This command lists the versions available for a Cloud Pod. Each invocation
  of the save command is going to create a new version for a named Cloud
  Pod, if a Pod with such name already does exist in the LocalStack
  platform.

Options:
  -f, --format [table|json]  The formatting style for the version command
                             output.

  --help                     Show this message and exit.
</disable-copy>
{{< / command >}}

The `versions` command lists all the available versions of a Cloud Pod. 
The CLI manual for the `version` command is as follows:

### `remote`

> The remote command lets you manage remotes different from the default one.
> It offers 3 commands, i,e.,: `add`, `delete`, and `list`.

Reviewer: ```suggestion
The `remote` command group lets you manage custom Cloud Pod remotes, to enable alternative storage backends in addition to the default LocalStack managed platform.
It offers 3 commands: `add`, `delete`, and `list`.
```

{{< command >}}
<disable-copy>
Usage: pod remote [OPTIONS] COMMAND [ARGS]...

  Manage cloud pod remotes

Options:
  --help  Show this message and exit.

Commands:
  add     Add a remote
  delete  Delete a remote
  list    Lists the available remotes
</disable-copy>
{{< / command >}}

#### `remote add`

{{< command >}}
<disable-copy>
Usage: pod remote add [OPTIONS] NAME URL

  Add a new remote for Cloud Pods.

  A remote is the place where your Cloud Pods are stored. By default, Cloud
  Pods are store in the LocalStack platform.

Options:
  --help  Show this message and exit.
</disable-copy>
{{< / command >}}

#### `remote delete`

{{< command >}}
<disable-copy>
Usage: pod remote delete [OPTIONS] NAME

  Remove a remote for Cloud Pods.

Options:
  --help  Show this message and exit.
</disable-copy>
{{< / command >}}

#### `remote list`
{{< command >}}
<disable-copy>
Usage: pod remote list [OPTIONS]

Options:
  -f, --format [table|json]  The formatting style for the remotes command
                             output.

  --help                     Show this message and exit.
</disable-copy>
{{< / command >}}

---

# Local Commands

> In alternative to the commands in the `pod` group, we also offer a simple alternative to save and load the LocalStack state.

Reviewer: ```suggestion
In addition to the commands in the `pod` group, we also offer a simple alternative to save and load the LocalStack state.
```

> The `state` group offers two commands to export and import the state from the LocalStack container to/from a zip file from the host machine.

Reviewer: ```suggestion
The `state` group offers two commands to export and import the state of the LocalStack container to/from a zip file from the host machine.
```

## `state` syntax
{{< command >}}
<disable-copy>
Usage: state [OPTIONS] COMMAND [ARGS]...

  (Beta) Manage and manipulate the localstack state.

  The state command group allows you to interact with LocalStack's state
  backend.

>   Read more: https://docs.localstack.cloud/references/persistence-
>   mechanism/#snapshot-based-persistence

Reviewer: I understand that the CLI auto-wraps these long lines, but we could consider printing them out in full here..
```suggestion
  Read more: https://docs.localstack.cloud/references/persistence-mechanism/#snapshot-based-persistence
```
